lib/send_publish.c

```diff
@@ -64,8 +64,7 @@ int send__publish(struct mosquitto *mosq, uint16_t mid, const char *topic, uint3
 #ifdef WITH_BROKER
 #ifdef WITH_BRIDGE
 	if(mosq->bridge && mosq->bridge->topics && mosq->bridge->topic_remapping){
-		for(i=0; i<mosq->bridge->topic_count; i++){
```

Comment:
Could you remove `int i` on line 48 now it's no longer in use please?

Reply:
Done